src/core/dfn.js

```diff
@@ -17,6 +17,12 @@ export function run() {
       dfn.dataset.dfnType = "idl";
     }
 
+    // Make it possible to reach dfns by tabbing, allowing keyboard actions as needed.
+    dfn.tabIndex = 0;
```

Comment:
Let's add tabIndex and title etc. in dfn-panel.js. Will help if we want to add an option to disable the feature in future, and keeps related code together.

Reply:
Agree about moving the title attribute, but I'm conflicted about tabindex. I see tabbing the dfn's as independent of the dfn panel. The dfn-panel just happens to respond to the keyboard activation of this element type.

Reply:
I don't think there are any other reasons why `<dfn>` should be focusable.

Reply:
I'd still like to keep significant modifications to dfn elements in dfn.js. Feels more natural having it here.

Reply:
Ah, forgot that Indexed terms also have popup menu (and they are `<span>`!)... so relocating this code to dfn-panel.

Reply:
<img width="507" alt="Screen Shot 2021-04-19 at 4 25 13 pm" src="https://user-images.githubusercontent.com/870154/115190780-e4d8a100-a12b-11eb-9073-f0be436a7445.png">

Now:

<img width="433" alt="Screen Shot 2021-04-19 at 4 27 26 pm" src="https://user-images.githubusercontent.com/870154/115190977-236e5b80-a12c-11eb-9b70-f0ee74616076.png">

Reply:
Why role=link?

Reply:
See first point in https://github.com/w3c/respec/issues/2825#issuecomment-811516805